core/tests/protractor/embedding.js

```diff
@@ -71,86 +124,106 @@ describe('Embedding', function() {
 
     users.createUser('user1@embedding.com', 'user1Embedding');
     users.login('user1@embedding.com', true);
-    adminPage.reloadExploration('protractor_test_1.yaml');
 
-    general.openEditor('12');
-    editor.setContent(forms.toRichText('Version 2'));
-    editor.saveChanges('demonstration edit');
+    // Create exploration.
+    // version 1 is creation of the exploration.
+    workflow.createExploration();
+    general.getExplorationIdFromEditor().then(function(expId){
+      explorationId = expId;
+      // creates version 2 of the exploration.
```

Comment:
`Create version 2 of the exploration.`

Start with a capital letter and be consistent with singular/plural tense.

Reply:
done.